features/support/world_extensions.rb

```diff
@@ -52,16 +52,31 @@ module MakesApiRequests
   def api_get(*args)
     set_auth_token
     get(*args)
+  rescue ActiveRecord::RecordNotFound
+    mock_404_response
```

Comment:
Another way to resolve this would be to make this change in config/environments/test.rb:

```diff
   # Raise exceptions instead of rendering exception templates.
-  config.action_dispatch.show_exceptions = false
+  config.action_dispatch.show_exceptions = true
```

but that might be undesirable for unit tests. An argument in favor would be that it'd make our test environment closer to our production environment, but that might be an argument for having a third environment, or for somehow running cucumber tests in the production environment instead of the test one.

Reply:
We should be able to tag the scenario with `@allow-rescue`, which will have the effect of changing this value to true for just that scenario -- letting Rails rescue the error and render the 404 template.

Reply:
https://github.com/cucumber/cucumber-rails/blob/5cf837c068214c240c39f74f513deef03a69d9db/features/allow_rescue.feature
https://github.com/cucumber/cucumber-rails/blob/8b162dcd6d27c59cf438d580384acac3f1f183d5/lib/cucumber/rails/hooks/allow_rescue.rb

Reply:
That worked! Cool.